api/v1alpha1/helpers.go

```diff
@@ -95,3 +95,11 @@ func (c *Contour) EnvoyTolerationsExist() bool {
 
 	return false
 }
+
```

Comment:
I think `ContourSecurityContextExists` and `EnvoySecurityContextExists` should have a comment line as these are public functions. I wonder why lint check does not produce an error :thinking: